docs/notifications.md

```diff
@@ -65,7 +69,46 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
 
 ## GitLab merge request comments
 
-Interested in GitLab merge request comments? [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)
+Merge request comments are currently supported for gitlab.com users only. [Sign up for the GitLab beta](https://go.r2c.dev/join-gitlab-beta) if you have a self-hosted instance of GitLab and are interested in receiving MR comments.
+
+Automated comments on GitLab merge requests look like this:
+
+<p style="text-align: center; font-size: 12px">
+    <img width="600px" src="../img/gitlab-mr-comment.png" alt="Screenshot of a GitLab MR comment"/><br/>
+    An inline GitLab merge request comment left by a custom Semgrep rule
+</p>
+
+To enable MR comments:
+
+1. Log into Semgrep's [Dashboard > Settings](https://semgrep.dev/manage/settings) to obtain your deployment ID and an API token.
+2. Examine your [policy settings](https://semgrep.dev/manage/policies/starter-policy?tab=settings) and make sure that PR comments are enabled. PR comments are the same thing as MR comments.
```

Comment:
```suggestion
2. Examine your [policy settings](https://semgrep.dev/manage/policies/starter-policy?tab=settings) and make sure that PR comments are enabled (MR comments and PR comments are enabled by the same toggle switch).
```

Note: this link didn’t load for me (I got a spinner but it never updated the main body of the page): https://semgrep.dev/manage/policies/starter-policy?tab=settings

Reply:
@pabloest yes, this is a tough judgment call. New users get a policy called "starter-policy" but older users will have "getting-started" or something similar. We could instead here just link to semgrep.dev/manage/policies, which will work for everyone, but it won't bring them right to the policy settings page - they have to then click an individual policy and navigate to the "settings" tab.

Reply:
@chmccreery ah makes sense. I’d leave the starter-policy then.

Reply:
In other words, I am happy to either stick with the link above, which will work for new users and not work for old users, or use /manage/policies, which will work for everyone but require two more clicks that may not be super obvious, so require more instructions here probably.

Reply:
I’m in favor of optimizing for new users by keeping the link you’re using in this PR.

Reply:
Oops, somehow didn't see your comments before now! Feel free to change back!! Sorry @pabloest

Reply:
Oops, somehow didn't see your comments before now! Feel free to change back!! Sorry @pabloest

Reply:
https://github.com/returntocorp/semgrep-docs/pull/214